server/handlers/component_handler.go

```diff
@@ -98,7 +109,17 @@ func (h *Handler) GetMeshmodelComponentsByName(rw http.ResponseWriter, r *http.R
 		ModelName: typ,
 		Version:   v,
 	})
-	if err := enc.Encode(res); err != nil {
+	var comps []v1alpha1.ComponentDefinition
+	for _, r := range res {
+		m := make(map[string]interface{})
+		comp, _ := r.(v1alpha1.ComponentDefinition)
+		_ = json.Unmarshal([]byte(comp.Schema), &m)
+		m = k8s.Format.Prettify(m, false)
```

Comment:
I see all the Format.Prettify() set to `false`, but don't see a `k8s.Format.Prettify(m, true)`. Should there be one?

Reply:
No. False here means the disablement of terminal value prettification

Reply:
Oh, I see. OK.